### PR TITLE
Updates to DAO api field names etc

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Just opening this for visibility - very much WIP at the moment

### CODE REVIEW

**Comment by seamuslee001** [comment]
@eileenmcnaughton can you rebase this now?

**Comment by eileenmcnaughton** [comment]
Done!

File: Readme.md [hunk 1]
```diff
@@ -2,14 +2,20 @@
...
+
+Third party testing and feedback are encouraged.
+
 ## Installation
 
 You can install this from github by doing:
 
     cd [your civicrm extensions directory]
-    git clone git@github.com:australiangreens/au.org.greens.extendedmailingstats
+    git clone git@github.com:mc0e/civicrm-extendedmailingstats.git au.org.greens.extendedmailingstats
```
Comment (seamuslee001) [inline Readme.md]: @eileenmcnaughton this needs to be updated to australiangreens/  etc

Comment (eileenmcnaughton) [reply Readme.md]: fixed

**Comment by eileenmcnaughton** [comment]
Houston we have merge-in

**Pull request merged**